src/Agda2Hs/Compile/Type.hs

```diff
@@ -131,16 +116,21 @@ compileTopLevelType keepType t cont = do
           underAbstraction a atel $ \tel ->
             go onTop tel (cont . qualifier)
 
-compileType' :: Term -> C (Strictness, Hs.Type ())
-compileType' t = do
+
+-- | Compile an Agda term into a Haskell type, along with its strictness.
+compileTypeStrictness :: Term -> C (Strictness, Hs.Type ())
```

Comment:
The name `compileTypeStrictness` is misleading, as it suggests *only* the strictness is "compiled'. Better use `compileTypeWithStrictness`?